Makefile

```diff
@@ -37,5 +37,11 @@ docker-sidecar:
 docker-testground:
 	docker build -t iptestground/testground:edge -f Dockerfile.testground .
 
+clean:
```

Comment:
`make test` might also fail with outdated `plans` in TESTGROUND_HOME... if this happens a user could just do `make clean` and then `make test`, which should always work.